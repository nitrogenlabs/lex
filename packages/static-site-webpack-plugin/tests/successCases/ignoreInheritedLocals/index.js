module.exports = (locals, callback) => {
  callback(locals.template({html: `<h1>"DONT_INCLUDE_ME" is in locals: ${'DONT_INCLUDE_ME' in locals}</h1>`}));
};
