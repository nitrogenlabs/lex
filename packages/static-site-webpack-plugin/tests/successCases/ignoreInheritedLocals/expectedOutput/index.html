<html>
  <body>
    <h1>"DONT_INCLUDE_ME" is in locals: false</h1>
  </body>
</html>
